### ISSUE TITLE
error at the convergence step

### ISSUE THREAD

**Issue by vaibhavadixit** [seekrcentral/seekr2#22]
Hi,
On the A4000 workstation that I mentioned earlier, the run.py step has completed successfully.
Then I tried the converge.py step but got the following error.
I'm somewhat surprised to see this error. It did run both BD and MD parts, thus can't understand why it is giving this error now.
Can you please suggest what might be wrong here?
thank you.

(base) [xxxx@xxxx seekr2]$ python3.8 converge.py /home/niperg/tryp_ben_hidr_tutorial/model.xml
Traceback (most recent call last):
  File "/home/niperg/miniconda3/lib/python3.8/site-packages/seekr2-0+untagged.226.g094afe7-py3.8.egg/seekr2/modules/elber_base.py", line 15, in <module>
    import openmm
ModuleNotFoundError: No module named 'openmm'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "converge.py", line 13, in <module>
    import seekr2.modules.common_converge as common_converge
  File "/home/niperg/miniconda3/lib/python3.8/site-packages/seekr2-0+untagged.226.g094afe7-py3.8.egg/seekr2/modules/common_converge.py", line 19, in <module>
    import seekr2.analyze as analyze
  File "/home/niperg/miniconda3/lib/python3.8/site-packages/seekr2-0+untagged.226.g094afe7-py3.8.egg/seekr2/analyze.py", line 21, in <module>
    import seekr2.modules.check as check
  File "/home/niperg/miniconda3/lib/python3.8/site-packages/seekr2-0+untagged.226.g094afe7-py3.8.egg/seekr2/modules/check.py", line 57, in <module>
    import seekr2.modules.elber_base as elber_base
  File "/home/niperg/miniconda3/lib/python3.8/site-packages/seekr2-0+untagged.226.g094afe7-py3.8.egg/seekr2/modules/elber_base.py", line 17, in <module>
    import simtk.openmm as openmm
  File "/home/niperg/miniconda3/lib/python3.8/site-packages/simtk/__init__.py", line 1, in <module>
    import openmm
ModuleNotFoundError: No module named 'openmm'

**Comment by lvotapka** [comment]
Are you sure that the correct conda environment and python version are being used to run converge.py? Try:

```
$ python3.8
>>> import openmm
```

**Comment by vaibhavadixit** [comment]
Hi,
Strangely I get an import error trying this.
Nonetheless, openmm is installed in miniconda3 directory as seen below and seekr2 commands did infact ran the GPU MD and CPU BD simulations since all anchor_0 ... folders have dcd and other files

Please do suggest what is wrong with my seerk2 installation on this workstation.
thank you for your continued suggestions and support. Best regards. 
Vaibhav

(base) [niperg@kaberi ~]$ python3.8
Python 3.8.13 (default, Mar 28 2022, 11:38:47)
[GCC 7.5.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import openmm
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'openmm'
>>> quit()

(base) [niperg@kaberi ~]$ find . -name openmm
find: ‘./.cache/gnome-control-center’: Permission denied
find: ‘./.config/gnome-control-center’: Permission denied
./.local/share/Trash/files/openmm-master/olla/include/openmm
./.local/share/Trash/files/openmm-master/openmmapi/include/openmm
./.local/share/Trash/files/openmm-master/platforms/common/include/openmm
./.local/share/Trash/files/openmm-master/plugins/amoeba/openmmapi/include/openmm
./.local/share/Trash/files/openmm-master/plugins/amoeba/serialization/include/openmm
./.local/share/Trash/files/openmm-master/plugins/drude/openmmapi/include/openmm
./.local/share/Trash/files/openmm-master/plugins/drude/serialization/include/openmm
./.local/share/Trash/files/openmm-master/plugins/rpmd/openmmapi/include/openmm
./.local/share/Trash/files/openmm-master/serialization/include/openmm
./.local/share/Trash/files/openmm-master/wrappers/python/openmm
./.local/share/Trash/files/openmm-master/wrappers/python/simtk/openmm
./Downloads/seekr2/seekr_tut/openmm
./Downloads/seekr2/seekr_tut/openmm/olla/include/openmm
./Downloads/seekr2/seekr_tut/openmm/openmmapi/include/openmm
./Downloads/seekr2/seekr_tut/openmm/platforms/common/include/openmm
./Downloads/seekr2/seekr_tut/openmm/plugins/amoeba/openmmapi/include/openmm
./Downloads/seekr2/seekr_tut/openmm/plugins/amoeba/serialization/include/openmm
./Downloads/seekr2/seekr_tut/openmm/plugins/drude/openmmapi/include/openmm
./Downloads/seekr2/seekr_tut/openmm/plugins/drude/serialization/include/openmm
./Downloads/seekr2/seekr_tut/openmm/plugins/rpmd/openmmapi/include/openmm
./Downloads/seekr2/seekr_tut/openmm/serialization/include/openmm
./Downloads/seekr2/seekr_tut/openmm/wrappers/python/openmm
./Downloads/seekr2/seekr_tut/openmm/wrappers/python/simtk/openmm
./Downloads/seekr2/seekr_tut/openmm/build/python/openmm
./Downloads/seekr2/seekr_tut/openmm/build/python/simtk/openmm
./miniconda3/pkgs/openmm-7.7.0-py38hb15d7a2_1/share/openmm
./miniconda3/pkgs/openmm-7.7.0-py38hb15d7a2_1/lib/python3.8/site-packages/simtk/openmm
./miniconda3/pkgs/openmm-7.7.0-py38hb15d7a2_1/lib/python3.8/site-packages/openmm
./miniconda3/pkgs/openmm-7.7.0-py38hb15d7a2_1/include/openmm
./miniconda3/pkgs/parmed-3.4.3-py310hd8f1fbe_2/lib/python3.10/site-packages/parmed/openmm
./miniconda3/lib/python3.8/site-packages/ParmEd-3.4.3-py3.8-linux-x86_64.egg/parmed/openmm
./miniconda3/lib/python3.8/site-packages/parmed/openmm
./miniconda3/share/openmm
./miniconda3/include/openmm
./miniconda3/envs/myambertools/lib/python3.10/site-packages/parmed/openmm

(base) [niperg@kaberi ~]$ pwd
/home/niperg
(base) [niperg@kaberi ~]$

**Comment by lvotapka** [comment]
I'm currently on vacation, but the most likely possible reason for this problem is because a different conda environment, machine, or user is running converge.py than ran run.py.

**Comment by lvotapka** [comment]
Hi @vaibhavadixit, did this error get resolved?

**Comment by vaibhavadixit** [comment]
Unfortunately, no.
I tried to install everything from scratch in a new environment but each time I'm getting a new error during the seekr2 test step.
Often it complains about seekrtools_plugin missing even after the seekr2_openmm_plugin is already installed and I can find a seekrtools_plugin inside the seekr2_openmm_plugin directory.
I did even try using the full procdure given [here](https://seekr2.readthedocs.io/en/latest/installation.html), but in vain.
In short, I'm finding installation very confusing.

I'm thinking of moving to Ubuntu on the workstation which you mentioned last time has fewer issues with seekr2.
Please suggest.
Thank you.

**Comment by lvotapka** [comment]
There is no plugin for seekrtools, so are you sure that it complains that "seekrtools_plugin" is missing? Did you try installing both seekr2 and seekrtools using "python setup.py install"?

Try skipping past the test step, and see what specific errors you encounter.

**Comment by lvotapka** [comment]
Closing this issue for now...
